scenario/LiquidationBotScenario.ts

```diff
@@ -64,7 +64,7 @@ for (let i = 0; i < MAX_ASSETS; i++) {
     {
       filter: async (ctx) => ctx.world.base.network === 'mainnet' && await isValidAssetIndex(ctx, i),
       tokenBalances: {
-        $comet: { $base: 10000 },
+        $comet: { $base: 3000 },
```

Comment:
we're getting `NotForSale()` errors in these scenarios, since the amount of `$base` specified previously (10K) is greater than the new `targetReserves` value (5k).